src/mlpack/methods/dbscan/dbscan_impl.hpp

```diff
@@ -225,17 +222,16 @@ void DBSCAN<RangeSearchType, PointSelectionPolicy>::PointwiseCluster(
  * naive search).
  */
 template<typename RangeSearchType, typename PointSelectionPolicy>
-template<typename MatType>
 void DBSCAN<RangeSearchType, PointSelectionPolicy>::BatchCluster(
     const MatType& data,
     UnionFind& uf)
 {
   // For each point, find the points in epsilon-neighborhood and their distances.
   std::vector<std::vector<size_t>> neighbors;
-  std::vector<std::vector<double>> distances;
+  std::vector<std::vector<ElemType>> distances;
   Log::Info << "Performing range search." << std::endl;
   rangeSearch.Train(data);
-  rangeSearch.Search(Range(0.0, epsilon), neighbors, distances);
+  rangeSearch.Search(RangeType<ElemType>(ElemType(0.0), epsilon), neighbors, distances);
```

Comment:
```suggestion
  rangeSearch.Search(RangeType<ElemType>(ElemType(0.0), epsilon), neighbors,
      distances);
```
This line got a little too long, this wraps it back to 80 characters.